vault/resource_ldap_auth_backend.go

```diff
@@ -67,6 +67,11 @@ func ldapAuthBackendResource() *schema.Resource {
 			Optional: true,
 			Computed: true,
 		},
+		"max_page_size": {
+			Type:     schema.TypeInt,
+			Default:  -1,
```

Comment:
If the Vault API returns a default for this field then we should not set a Default in the provider. We can instead set this field to `Computed: true`.

Reply:
The issue is that currently Vault API returns the wrong value (0) instead of (-1), which breaks most LDAP implementations, as it causes no results to be sent. They have their own PR with a fix being made, but this change makes it backwards compatible.

https://developer.hashicorp.com/vault/api-docs/auth/ldap#max_page_size

Reply:
ok, thanks for the explanation. I think in this case we can set the Default then.